chess-AI/train.py

```diff
@@ -235,7 +237,7 @@ def create_stockfish_dataset(sf_opt, show_dash):
     return create_dataset(sf_opt.games, stocktrain_moves, stocktrain_value_approximator, show_dash)
 
 
-def train_on_mcts(nnet, mcts_opt, show_dash=False):
+def train_on_mcts(nnet, mcts_opt, save_freq=10, show_dash=False):
```

Comment:
Like you said in the PR description, `save_freq` should be part of ai_io (and the params file/dashboard). since we're not working on the dashboard rn don't worry about it adding it there, but pls add it to the params file and update init_params. Can you also add `save_freq` as a field in `mcts_opt`? Then we don't need to update the function signature